server/src/main/resources/db/common/V107__create_volunteering_tables.sql

```diff
@@ -16,7 +16,7 @@ CREATE TABLE volunteering_relationship
     relationship_id varchar PRIMARY KEY,
     member_id       varchar REFERENCES member_profile (id),
     organization_id varchar REFERENCES volunteering_organization (organization_id),
-    start_date      timestamp NOT NULL,
```

Comment:
You cannot edit existing migrations. You have to create a new migration (with the next version number) and alter the table instead of changing this one.

Reply:
This will be fixed momentarily.